engine/lib/upgrades/2015062400-1.11.2-discussions_plugin-92f374deba4dc416.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Elgg 1.11.2 upgrade 2015062400
+ * discussions_plugin
+ *
+ * Discussions feature was pulled from groups plugin into its
+ * own plugin, so we need to update references of subtype
+ * 'groupforumtopic' into 'discussion'.
```

Comment:
What is the gain we get from changing the subtype? Because it seems like a big cost to BC.